x-pack/plugins/security_solution/public/timelines/containers/local_storage/index.tsx

```diff
@@ -85,13 +85,20 @@ export const addTimelineInStorage = (
   id: TimelineIdLiteral,
   timeline: TimelineModel
 ) => {
+  const timelineToStore = cleanStorageTimeline(timeline);
   const timelines = getAllTimelinesInStorage(storage);
   storage.set(LOCAL_STORAGE_TIMELINE_KEY, {
     ...timelines,
-    [id]: timeline,
+    [id]: timelineToStore,
   });
 };
 
+const cleanStorageTimeline = (timeline: TimelineModel) => {
+  // clean filterManager as it contains circular references within observer subscriptions
+  const { filterManager, ...timelineToStore } = timeline;
```

Comment:
I think that we need to remove all of these attributes to be like it was in 7.13. The attribute `defaultModel` is now `defaultColumn`, we simplified that.

![image](https://user-images.githubusercontent.com/189600/131596318-018df5b3-e468-4cac-858f-f9e8f269f77c.png)

https://github.com/elastic/kibana/blob/7.13/x-pack/plugins/security_solution/public/timelines/components/manage_timeline/index.tsx#L31-L43